### DISCUSSION TITLE
求个跑京东云无限宝的配置-pcndn

### DISCUSSION THREAD

**Discussion by yuyi4511** [coolsnowwolf/lede#9571] [R68S / 电犀牛专区]
r68s为京东云的上级，怎么设置让上传跑起来。
是固件问题还是规则设置的问题，

**Comment by ljlwdf** [comment]
我的也跑不起来，应该是固件问题吧。也没什么设置的呀

**Comment by a641764137** [comment]
我用lean大的固件都跑不起来，小宝的istore os没问题，但是不能装openclash。

**Comment by ljlwdf** [comment]
已经知道原因了，关闭“SFE 加速”就好了。

**Reply by zy8236** [reply]
大佬，关掉sfe加速后影响性能吗

**Comment by shuimulingyang** [comment]
有没有多拨跑分能上去的

**Comment by wenlin5945** [comment]
我的也跑不起来，我现在把r68s坐二级路由了
